sdk/confidentialledger/confidential-ledger-rest/README.md

```diff
@@ -250,6 +291,30 @@ const getLedgerEntriesParams = { queryParameters: { collectionId: "my collection
 const ledgerEntries = await client.path("/app/transactions").get(getLedgerEntriesParams);
 ```
 
+### Get Transactions for a Collection by CollectionId and Tags
+
+```ts snippet:ReadmeSampleGetTransactionsForCollectionAndTags
+import ConfidentialLedger, { getLedgerIdentity } from "@azure-rest/confidential-ledger";
+import { DefaultAzureCredential } from "@azure/identity";
+
+const { ledgerIdentityCertificate } = await getLedgerIdentity(
+  "test-ledger-name",
+  "https://identity.confidential-ledger.core.azure.com",
+);
+const credential = new DefaultAzureCredential();
+
+const client = ConfidentialLedger(
+  "https://test-ledger-name.confidential-ledger.azure.com",
+  ledgerIdentityCertificate,
+  credential,
+);
+
+const getLedgerEntriesParams = {
+  queryParameters: { collectionId: "my collection", tags: "tag1" },
```

Comment:
This wont work due to typo in parameter.

Please download the sdk, install it and verify the code before trying to merge it.